eng/native/functions.cmake

```diff
@@ -401,10 +401,12 @@ function(strip_symbols targetName outputFilename)
 
       set(strip_command ${STRIP} -no_code_signature_warning -S ${strip_source_file})
 
-      # codesign release build
-      string(TOLOWER "${CMAKE_BUILD_TYPE}" LOWERCASE_CMAKE_BUILD_TYPE)
-      if (LOWERCASE_CMAKE_BUILD_TYPE STREQUAL release)
-        set(strip_command ${strip_command} && codesign -f -s - ${strip_source_file})
+      if (CLR_CMAKE_TARGET_OSX)
```

Comment:
Perhaps just:
```suggestion
      if (NOT CLR_CMAKE_TARGET_MACCATALYST)
```
?

Reply:
the only other targets are iOS/tvOS and it doesn't make much sense to codesign there either since that will be done  during the app build. we've also not done this in the mono.proj version before you change so I think it's fine to limit it to OSX.